### PR TITLE
Frontpage facelift

### PR DESCRIPTION
**Pull request by kristjanmik**
Better than what we currently have.

Start the project and go to localhost:3001 to see the frontpage

### CODE REVIEW

**Comment by tryggvigy** [comment]
Can you provide a screen shot?

File: src/routes/frontpage/backers.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    description: 'The travel transformation platform',
+  },
+];
+export const supporters = [
+  {
+    name: 'Northstack',
+    url: 'http://northstack.is',
+    logo: '/logos/northstack.png',
+    description: "Northstack is Iceland's premier outlet for news, data, analysis, and discussion about the Icelandic startup and tech scene.",
+  },
```
Comment (johannth) [inline src/routes/frontpage/backers.js:42]: There is something weird about the mix of Icelandic and English. Easy to fix later though.

File: src/routes/frontpage/Frontpage.js [hunk 2]
```diff
@@ -103,14 +109,14 @@ class Frontpage extends Component {
...
               <div style={{ flex: 8 }} />
 
-              <div className={s.footerItemWrapper}>
+              {/* <div className={s.footerItemWrapper}>
                 <a className={s.footerItemInner} href="https://grassroots.com/Press.zip">
                   <img className={s.pressIcon} src="/img/press@2x.png" alt="press" />
                   Press
                 </a>
-              </div>
+              </div> */}
```
Comment (johannth) [inline src/routes/frontpage/Frontpage.js]: Why not just delete?

File: src/routes/frontpage/Frontpage.js [hunk 3]
```diff
@@ -50,49 +71,34 @@ class Frontpage extends Component {
...
-                  <div className={s.clientTitle}>Founding Partner</div>
-                  <div className={s.clientText}>
-                    Tempo is an Icelandic company that builds productivity software
-                  </div>
-                </div>
-              </div>
-              <div className={s.clientImageContainer}>
-                <img className={s.clientImage} src="/img/tempo.png" alt="tempo" />
-              </div>
+            {/* <h1 className={s.backerCategory}>Operators</h1> */}
```
Comment (johannth) [inline src/routes/frontpage/Frontpage.js]: We not just delete?

File: src/routes/frontpage/Frontpage.css [hunk 4]
```diff
@@ -363,11 +348,13 @@
   cursor: pointer;
 }
 
+/*
 .pressIcon {
   width: 53px;
   height: 43px;
   margin-bottom: 14px;
 }
+*/
```
Comment (johannth) [inline src/routes/frontpage/Frontpage.css]: I'm a huge fan of delete, haha

**Review by johannth — APPROVED** [review]
Looking good, minor comments. Also there is a rouge .DS_STORE file commited. Maybe add that to .gitignore.

File: src/routes/frontpage/Frontpage.css (hunk 4 above)
Comment (tryggvigy) [reply src/routes/frontpage/Frontpage.css]: I'm with you. You can always go back in git history to find this code if it needs reintroduction

**Comment by johannth** [comment]
I'm going to push up minor cleanups and then merge.

**Pull request merged**